### DISCUSSION TITLE
iPad build

### DISCUSSION THREAD

**Discussion by iamwavecut** [zed-industries/zed#16353] [Feature Requests]
There is a whole empty market which is an iPadi IDE. 

iPad with magic keyboard is an amazing machine to practice remote (and even local if you're nerdy enough) development. 
But no one's taking it seriously and that works both ways for devs and apps creators. 

Thus said, I see and expreriense the lack of decent IDE for iPad, and i have no hope for VSCode now to support it, while web based VSCode is not usable due to it's extension host being pretty limited inside webview (no language servers, etc).

Are you considering releasing Zed for iPad? Fingers crossed

**Comment by bajrangCoder** [comment]
Currently there is no plan.
Btw here is issue for this https://github.com/zed-industries/zed/issues/12039
